addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -7404,6 +7404,7 @@ QUnit.module('basic_fields', {
     QUnit.test('FieldBadge component on a many2one field in list view', async function (assert) {
         assert.expect(2);
 
+        this.data.partner.records[0].trululu = 4;
```

Comment:
You found this test because it failed, I guess, but there might be other tests implicitly relying on this value, that do not fail, but that are no longer pertinent.

Reply:
If other tests implicitly rely on the value but they still work without it, that means they are weak tests in regard this value, and they should be improved to actually fail when the value is not there.

I can't guess what people expect and if such assumptions exist it would make the test suite unrefactorable/unmaintable.

Reply:
> I can't guess what people expect and if such assumptions exist it would make the test suite unrefactorable/unmaintable.

That's why we usually don't do such changes ^^